packages/language-server/src/plugins/svelte/features/getCodeActions/getQuickfixes.ts

```diff
@@ -207,26 +225,31 @@ function getSvelteIgnoreEdit(
     const indent = getIndent(afterStartLineStart);
 
     // TODO: Make all code action's new line consistent
-    const ignore = `${indent}<!-- svelte-ignore ${code} -->${EOL}`;
+    let ignore = `${indent}// svelte-ignore wtf ${code}${EOL}${indent}`;
```

Comment:
😄 
```suggestion
    let ignore = `${indent}// svelte-ignore ${code}${EOL}${indent}`;
```

Reply:
Hahahaha i was getting frustrated because changes were not reflecting during debugs 😄